app/src/main/java/org/oppia/app/player/state/audio/LanguageDialogFragment.kt

```diff
@@ -0,0 +1,69 @@
+package org.oppia.app.player.state.audio
+
+import android.app.Dialog
+import android.os.Bundle
+import androidx.appcompat.app.AlertDialog
+import androidx.fragment.app.DialogFragment
+import org.oppia.app.R
+import java.util.ArrayList
+
+/**
+ * DialogFragment that controls language selection in audio and written translations
+ */
+class LanguageDialogFragment : DialogFragment() {
+  companion object {
+    /**
+     * This function is responsible for displaying content in DialogFragment
+     * @param title: title of the Dialog
+     * @param languageCodeList: List of strings containing languages
+     * @param currentLanguageCode: Currently selected language code
+     * @return LanguageDialogFragment: DialogFragment
+     */
+    fun newInstance(
+      title: String,
+      languageCodeList: List<String>,
+      currentLanguageCode: String
+    ): LanguageDialogFragment {
+      val selectedIndex: Int = languageCodeList.indexOf(currentLanguageCode)
+      val frag = LanguageDialogFragment()
+      val args = Bundle()
+      args.putString("TITLE", title)
+      args.putStringArrayList(
+        "LANGUAGE__LIST",
```

Comment:
Prefer keeping bundle keys as top-level constants. Particularly because sometimes they can become out-of-date (I think you have a typo here that would cause prod code to break per retrieval below :) ).

Reply:
Done. created constants on top-level in this file.